sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -316,6 +316,20 @@ static int mt8195_dsp_suspend(struct snd_sof_dev *sdev, u32 target_state)
 {
 	struct platform_device *pdev = container_of(sdev->dev, struct platform_device, dev);
 	int ret;
+	int reset_sw, dbg_pc, sleep_try = 2000, sleep_cnt = 0;
+
+	/* wait 2 second to wait dsp enter wait for interrupt */
```

Comment:
The wording of the comment is not correct.

Is it really can take up to 2 seconds? That is awful long time to delay the suspend callback.